tests/gcp/bundle_test.go

```diff
@@ -1,321 +1,145 @@
-package test
+package gcp
 
 // Set PREFIX, GCP_PROJECT, and GOOGLE_APPLICATION_CREDENTIALS credentials before running these scripts
 
 import (
-    "testing"
-    "os"
-    "strconv"
-    "fmt"
-    "encoding/json"
-    "strings"
-    "time"
-
-    "github.com/gruntwork-io/terratest/modules/retry"
-    "github.com/gruntwork-io/terratest/modules/gcp"
-    "github.com/gruntwork-io/terratest/modules/terraform"
-    "github.com/gruntwork-io/terratest/modules/ssh"
-    "github.com/stretchr/testify/assert"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/gcp"
+	"github.com/gruntwork-io/terratest/modules/ssh"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/protofire/polkadot-failover-mechanism/tests/helpers"
+	"github.com/stretchr/testify/assert"
 )
 
 //Gather environmental variables and set reasonable defaults
-var gcpRegion = [3]string{"us-east1", "us-east4", "us-west1"}
-var prefix = os.Getenv("PREFIX")
+var (
+	gcpRegion  = []string{"us-east1", "us-east4", "us-west1"}
+	gcpProject = os.Getenv("GCP_PROJECT")
+	sshUser    = "ubuntu"
+)
 
 func TestBundle(t *testing.T) {
 
-    // Generate new SSH key for test virtual machines
+	var (
+		prefix string
+		ok     bool
+	)
+
+	if prefix, ok = os.LookupEnv("PREFIX"); !ok {
+		prefix = "test"
+	}
+
+	// Generate new SSH key for test virtual machines
 	sshKey := ssh.GenerateRSAKeyPair(t, 4096)
 
-	// Configure Terraform - set backend, minimum set of infrastructure variables. Also expose ssh 
+	// Configure Terraform - set backend, minimum set of infrastructure variables. Also expose ssh
 	terraformOptions := &terraform.Options{
 		// The path to where our Terraform code is located
 		TerraformDir: "../../gcp/",
 
 		// Variables to pass to our Terraform code using -var options
 		Vars: map[string]interface{}{
-            "gcp_regions": "[\"" + gcpRegion[0] + "\", \"" + gcpRegion[1] + "\", \"" + gcpRegion[2] + "\"]",
-	    "gcp_project": os.Getenv("GCP_PROJECT"),
-            "validator_keys": "{key1={key=\"0x6ce96ae5c300096b09dbd4567b0574f6a1281ae0e5cfe4f6b0233d1821f6206b\",type=\"gran\",seed=\"favorite liar zebra assume hurt cage any damp inherit rescue delay panic\"},key2={key=\"0x3ff0766f9ebbbceee6c2f40d9323164d07e70c70994c9d00a9512be6680c2394\",type=\"aura\",seed=\"expire stage crawl shell boss any story swamp skull yellow bamboo copy\"}}",
-            "gcp_ssh_user": "ubuntu",
-            "gcp_ssh_pub_key": sshKey.PublicKey,
-            "prefix": prefix,
-            "delete_on_termination": "true",
-            "cpu_limit": "1",
-            "ram_limit": "1",
-	    "validator_name": "test",
-	    "expose_ssh": "true",
-	    "node_key": "fc9c7cf9b4523759b0a43b15ff07064e70b9a2d39ef16c8f62391794469a1c5e",
-            "chain": "westend",
-	    "admin_email": "1627_DEV@altoros.com", 
+			"aws_regions":           helpers.BuildRegionsParam(gcpRegion...),
```

Comment:
aws_regions in GCP? Probably a mistake here.

Reply:
Fixed